tasks/bootstrap_user.yml

```diff
@@ -16,10 +16,17 @@
 - name: Set bootstrap user as Galaxy Admin
   lineinfile: dest={{ galaxy_config_file }} state=present insertafter="app:main" regexp="^[ ]*admin_users[ ]*=[ ]*" line="admin_users = {{ galaxy_tools_admin_user }}"  #"
   when: (create_user is defined) and create_user and (admin_users_found.msg == "")
+  notify:
+    - restart galaxy
```

Comment:
Just out of interest is the renmaming of restart_galaxy needed to make this work?